scripts/fuzz_opt.py

```diff
@@ -1343,6 +1343,11 @@ def handle(self, wasm):
         if not ctors:
             return
 
+        # Fix escaping of the names, as we will be passing them as commandline
+        # parameters below (e.g. we want --ctors=foo\28bar and not
+        # --ctors=foo\\28bar; that extra escaping \ would cause an error).
+        ctors = bytes(ctors, 'utf-8').decode('unicode_escape')
```

Comment:
Where is the corresponding encoding done? Is 'unicode_escape` actually a match for it?

Reply:
`get_exports` parses wat text, which is encoded/escaped. This just undoes that.

Reply:
I see, thanks. I was hoping the escaping was all matching when I saw that this one issue worked perfectly :smile: 

Changed it to unescape the specific issue I am seeing with extra slashes, that we don't need.